topics/index.html

```diff
@@ -85,14 +85,14 @@ <h1 class="logotype">MIDAAS</h1></a>
             </li>
             <li class="topic coming-soon">
               <div class="container">
-                <h2>How Has Income Changed in My City?</h2>
-                <p class="description">Explore changes from 2009 to 2015.</p>
+                <h2>How has Income Changed in My City?</h2>
+                <p class="description">Inspect the economic growth in different cities from 2009 to 2015.</p>
```

Comment:
The changes here don't look right... I think this branch needs to be updated to the latest `develop`.